daisybell/scanners/ner_language_bias.py

```diff
@@ -0,0 +1,96 @@
+import pandas as pd
+import pysbd
+from tqdm.auto import tqdm
+from transformers import Pipeline
+
+from daisybell.daisybell import scanner
+from daisybell.helpers.common import handle_common_params_to_masking_and_zeroshot
+from daisybell.helpers.dataset import emit_books, replace_entities
+
+
+@scanner(
```

Comment:
Does the annotations work when in separate files? I had bad luck with this.

Try running daisybell from the command line eg.

`daisybell roberta-base`

Or on a GPU machine (better)

`daisybell roberta-base -d cuda`

If the command crashes or returns very fast without scanner output, it means none of the scanners registered.

Reply:
In the latest commits I switched everything over to a plugin based model. This now works with the exact usage pattern that you've described above.